content/en/docs/getting-started/setup/create-cluster/_index.md

```diff
@@ -12,14 +12,110 @@ aliases:
+
+The following two paragraphs define the various configuration variables as well as the Module's output variables.
+
+#### Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:-----:|
+| cluster\_name | Name of the K8s cluster to create | `string` | `"jenkins-x"` | no |
+| dev\_env\_approvers | List of git users allowed to approve pull request for dev enviornment repository | `list(string)` | `[]` | no |
+| force\_destroy | Flag to determine whether storage buckets get forcefully destroyed | `bool` | `false` | no |
+| gcp\_project | The name of the GCP project to use | `string` | n/a | yes |
+| git\_owner\_requirement\_repos | The git id of the owner for the requirement repositories | `string` | `""` | no |
+| max\_node\_count | Maximum number of cluster nodes | `number` | `5` | no |
+| min\_node\_count | Minimum number of cluster nodes | `number` | `3` | no |
+| node\_disk\_size | Node disk size in GB | `string` | `"100"` | no |
+| node\_machine\_type | Node type for the K8s cluster | `string` | `"n1-standard-2"` | no |
+| parent\_domain | The parent domain to be allocated to the cluster | `string` | `""` | no |
+| tls\_email | Email used by Let's Encrypt. Required for TLS when parent\_domain is specified. | `string` | `""` | no |
+| velero\_schedule | The Velero backup schedule in cron notation - check https://github.com/jenkins-x/jenkins-x-boot-config/blob/master/systems/velero-backups/templates/default-backup.yaml for defaults | `string` | `"0 * * * *"` | no |
+| velero\_ttl | The time allocated that defines the lifetime of a velero backup - check https://github.com/jenkins-x/jenkins-x-boot-config/blob/master/systems/velero-backups/templates/default-backup.yaml for defaults | `string` | `"720h0m0s"` | no |
+| version\_stream\_ref | The git ref for version stream to use when booting Jenkins X. See https://jenkins-x.io/docs/concepts/version-stream/ | `string` | `"master"` | no |
+| version\_stream\_url | The URL for the version stream to use when booting Jenkins X. See https://jenkins-x.io/docs/concepts/version-stream/ | `string` | `"https://github.com/jenkins-x/jenkins-x-versions.git"` | no |
+| webhook | Jenkins X webhook handler for git provider | `string` | `"prow"` | no |
+| zone | Zone in which to create the cluster | `string` | `"us-central1-a"` | no |
+
+#### Outputs
+
+| Name | Description |
+|------|-------------|
+| backup\_bucket\_url | The URL to the bucket for backup storage |
+| cluster\_name | The name of the created K8s cluster |
+| gcp\_project | The GCP project in which the resources got created in |
+| log\_storage\_url | The URL to the bucket for log storage |
+| report\_storage\_url | The URL to the bucket for report storage |
+| repository\_storage\_url | The URL to the bucket for artefact storage |
+| vault\_bucket\_url | The URL to the bucket for secret storage |
+| zone | The zone of the created K8s cluster |
+
+
+## Amazon Elastic Kubernetes Service (EKS)
```

Comment:
@dgozalo - here I thought we plugin whatever is relevant for EKS Terraform. WDYT?
What do you think about the navigation from "Getting started" to here?